lwc/src/main/scala/quasar/physical/s3/S3LWFS.scala

```diff
@@ -26,12 +26,27 @@ import org.http4s.client.Client
 import org.http4s.Uri
 import scalaz.concurrent.Task
 
+// three functions comprise the LightweightFileSystem API
+// (and informally, the lightweight connector API itself)
+// since they're all implemented in their own files and
+// delegated to, I'll focus on the interface itself.
```

Comment:
The comments on this class and the functions below could be scaladoc formatted.

Also since this is user-facing doc, capital letters at the starts of sentences would be more professional.

Reply:
Not sure what you're getting at with the last two lines in this comment. Maybe just remove them?